wazo_load_api/wlapi/docker/Dockerfile.tmpl

```diff
@@ -85,7 +85,7 @@ COPY --from=builder /usr/src/wlapi /usr/src/wlapi
 COPY --from=builder /etc /etc
 COPY wlapi/docker/.baresip /root/.baresip
 COPY wlapi/docker/docker-entrypoint.sh /docker-entrypoint.sh
-COPY wlapi/docker/Rameses.wav /opt/Rameses.wav
+RUN wget https://gist.github.com/pc-m/7bc089a402a89ad00ec5323ca909934d/raw/b54c6e6521eeef282defef2d7a91df8dd093158e/Rameses.wav -O /opt/Rameses.wav
```

Comment:
It was originally on the filesystem of one of the servers in our internal infrastructure. I used a gist to avoid adding a 30MB binary to the git repo. Ideally I would want to host the file somewhere public under our control.